README.md

```diff
@@ -8,3 +8,13 @@ To build and run this application please use the `kraft` script:
 
 For more information about `kraft` type ```kraft -h``` or read the
 [documentation](http://docs.unikraft.org).
+# Dependencies
+To run duktape, a root filesystem must be mounted.
+To build, python 2.7 needs to be installed under the "python" name with pyYaml make sure you do not have a "python" to "python3". This can be done with the following bash commands:
+
+    sudo apt install python2
```

Comment:
Please do not use `python2`, since it is not supported anymore.

Instead, you may try and update the version of the application. You can check the latest versions [here](https://duktape.org/download.html). I would suggest going for 2.7 version of ducktape, since it supports `python3`.